src/benchmarks/real-world/Roslyn/Program.cs

```diff
@@ -15,12 +17,16 @@ public static async Task<int> Main(string[] args)
         {
             await Setup();
 
-            return BenchmarkSwitcher
-                .FromAssembly(typeof(Program).Assembly)
-                .Run(args, RecommendedConfig.Create(
-                    artifactsPath: new DirectoryInfo(Path.Combine(Path.GetDirectoryName(typeof(Program).Assembly.Location), "BenchmarkDotNet.Artifacts")),
-                    mandatoryCategories: ImmutableHashSet.Create("Roslyn")))
-                .ToExitCode();
+            var summary = BenchmarkRunner.Run<StageBenchmarks>(
```

Comment:
I know that as of today this project defines a single type with benchmarks, but I would rather prefer to keep using `BenchmarkSwitcher`. It supports more than 1 type (but it requires to provide the `--filter`  argument or choosing type from the list.)

Moreover, it supports parsing console line arguments, which are passed by the CI to the benchmark app. A good example is `--stopOnFirstError true` which stops running the benchmarks after first error. I am not 100% sure but most probably the CI failure is due to removing the `.Run(args` part